dev/Telemetry/RuntimeProfiler.h

```diff
@@ -43,6 +43,9 @@ namespace RuntimeProfiler
         ProfId_ItemsRepeaterScrollHost,
         ProfId_TabView,
         ProfId_TabViewItem,
+        ProfId_FlipView,
```

Comment:
Without the code coming over do you need these? I didn't add them for the other controls that are template-only.